addons/website_sale/static/src/snippets/s_dynamic_snippet_products/000.js

```diff
@@ -13,25 +13,13 @@ const DynamicSnippetProducts = DynamicSnippetCarousel.extend({
     // Private
     //--------------------------------------------------------------------------
 
-    /**
-     * Check product category exist.
-     * @private
-     * @returns {Promise}
-     */
-    _checkCategoryExist: async function (productCategoryId) {
-        return this._rpc({
-            model: 'product.public.category',
-            method: 'search_count',
-            args: [[['id', '=', productCategoryId]]],
-        }).then(nb => nb > 0);
-    },
     /**
      * Method to be overridden in child components if additional configuration elements
      * are required in order to fetch data.
      * @override
      * @private
      */
-    _isConfigComplete: function () {
+    _isConfigComplete: function() {
```

Comment:
I think this one can stay as function () is correct 🙂

Reply:
Done :+1: